Doc 3750 Updates to the Log4j release schedule (#478)
corda/corda-docs-portal: content/en/release-notes/_index.md

## Apache Log4j announcement

### Update December 20 2021

Investigations are in progress following the release of Log4j 2.17.0. As Corda’s explicit disabling of Java

> serialisation is an effective countermeasure against the vulnerabilities, the update to Log4j 2.17.0 (or the latest version

Reviewer: ```suggestion
serialization is an effective countermeasure against the vulnerabilities, the update to Log4j 2.17.0 (or the latest version
```

## Apache Log4j announcement

### Update December 20 2021

Investigations are in progress following the release of Log4j 2.17.0. As Corda’s explicit disabling of Java
serialisation is an effective countermeasure against the vulnerabilities, the update to Log4j 2.17.0 (or the latest version

> at that time) will be available in xxx.

Author: ```suggestion
at that time) will be available end of January 2022.
```

> CENM 1.5.3, CENM 1.3.4, CENM management console (Gateway Plugin), and Business Network Manager tool 1.1.1 updates have

Author: ```suggestion
CENM 1.3.4, CENM management console (Gateway Plugin), and Business Network Manager tool 1.1.1 updates have
```

| Corda OS/CE 4.3.10     | **Released** Dec 16        | Yes                          |
| CENM 1.2.5             | **Released** Dec 17        | No                           |

> | CENM 1.5.3             | Dec 20                     | Yes                          |

Author: ```suggestion
| CENM 1.5.3             | Dec 21                     | Yes                          |
```
